http_client.go

```diff
@@ -63,6 +63,9 @@ func DoRequest(params *RequestParams) (*HTTPMessage, error) {
 	if params.NoAutoHeaders {
 		headers = params.Headers
 	} else {
+		if params.Target.Path == ""{
```

Comment:
Hello,

1. Please apply `go fmt` to this line.
2. What is the case you're addressing (=when the Target.Path is empty)?

Reply:
It’s just that I encountered that it was not detected in `https://example.com`, but `https://example.com/`